### ISSUE TITLE
As a user, I want to be able to reject updated AUPs, and clearly see the consequences for CO membership if I do so

### ISSUE THREAD

**Issue by FlorisFokkinga** [SURFscz/SBS#664]
The redesign for the changed AUP screen.
The table with inaccessible collaborations should change based on the checked boxes.

### First screen ###
![Image](https://user-images.githubusercontent.com/80269262/233985763-5e5c195b-a773-4001-98e2-96b3047400bf.png)

### Modal in case not all AUPs are accepted ###
![Image](https://user-images.githubusercontent.com/80269262/233985776-038af89c-9620-4208-8b78-60c80a8869da.png)

**Comment by FlorisFokkinga** [comment]
## The checkbox only applies to the AUPs, not to the privacy policies. ##

**Comment by oharsta** [comment]
@FlorisFokkinga @baszoetekouw  If a User reject an AUP, then he / she will be asked to agree every time he / she accesses SBS and also if he tries to connect directly to the service because of the login in the `proxy_authz` endpoint logic called by eduTeams. This is the desired behaviour?

**Comment by baszoetekouw** [comment]
I think that if a user rejects the AUP, they need to be removed from the CO.  So they would only get the AUP question again if they rejoin the CO, I think?

**Comment by oharsta** [comment]
Ok. I assume we also need to change the attached screen that is shown when a user logs in and there are updated AUP's?
<img width="1240" alt="image" src="https://github.com/SURFscz/SBS/assets/530657/cd8faebf-e7d1-47b6-b41b-5f721c384c96">

**Comment by oharsta** [comment]
And do we need to change the welcome dialog shown if a user joins a CO?

![Image](https://github.com/SURFscz/SBS/assets/530657/8f7ef61f-3085-45fa-8fa5-cb7933663120)

**Comment by FlorisFokkinga** [comment]
@oharsta  You are right about the other screens. The plan was to first solve the complicated case of multiple reset AUPs over multiple COs. I expected that solution to also work for the simpler cases.
If you think it's better to fix those as well at the same time, and there's a logical solution for those cases, please feel free to do so. 🙂

**Comment by baszoetekouw** [comment]
@oharsta do you have enough information to implement this, or do we need to expand on the UI?

**Comment by oharsta** [comment]
Enough information. Just didn't get round to.

**Comment by logan-life** [comment]
In principle, this is a good thing to do and aligns with values around user autonomy and privacy. In practice, this is probably not something users actually care about. They will click the green button and get to work. 

Further -- in future, we will put effort into reducing the number of AUPs we throw at users, so adjusting our principles for AUPs. So for now, let's not put effort into this.

**Comment by logan-life** [comment]
Agree with both August 2024 Logan and December 2025 Logan. But, we will consider it.
